rules.md

```diff
@@ -425,274 +284,367 @@ The Meta % is an added commission that an Admin earns for processing a task clai
+
+## 4.5 Governance
 
-QA Rewards are set as a % added to the relevant Task the QA refers to, defined in the [Price List](#25-price-list).
+Admins are responsible for Governance of the Incubator which is implemented via appointing or removing Admins, or updating these Rules.
 
-QA Tasks claims should be requested by users and processed by Admins like regular Production tasks.  In cases where the QA provider is the Primary Admin, the Secondary Admin is required to approve the claim as with all tasks.
+An Admin Quorum is required in both cases, which means that >= 51% agree (or <=49% don't object) to Admin appointments / removals or changes to the Rules.
 
-### 4.3.4 Task Reservations
+# 5 Funds
 
-The Primary Admin can reserve unassigned tasks to users who have left a valid [Request Reservation](#233-reserving-a-task) comment.  If the Admin deems the request valid, they assign the user to the tasks in the Tasks checklist, which removes those tasks from availability on the Incubator website. They may also comment to alert the user that the relevant tasks were reserved, for example:
+## 5.1 Proposals
 
-```@username reserved tasks: 1,2,3,4,5```
+The App is funded from a series of [Proposals](#64-proposals) submitted to the [Dash Network](https://docs.dash.org/en/stable/governance/understanding.html) on a quarterly basis, with all spending of those funds commissioned and tracked via Tasks owned by Admins within the Bounty System itself and settled via transactions on the Dash blockchain that anyone can verify.
 
-If the comment is invalid, either technically or for another reason such as lack of reputation or some previous issue, the Admin should reply to the comment to request clarification or give the reason(s) why the request was denied, for example:
+The amount requested might vary to target the general scale and buffer that Admins and the [PO](#53-proposal-owner) feel the Incubator should aim to spend, but not to deliver specific projects or milestones as there is no centralized planning to do so in the Incubator.
 
-```@username, there are two "task 3"s, are you requesting to reserve Specification Task 3, or Production Task 3?```
+When certain projects arise that do require specific additional funding, individual proposals should be raised detailing the value proposition and estimated costs of those projects individually.
 
-### 4.3.5 Task Claims
+## 5.2 Budgets
 
-When an Admin wants to approve a user’s Claim comment, they add a Claim Approved comment as follows:
+All funds will be received to the published [Incubator Wallet](#63-accounting) address(s) and will be pooled across proposals as a single rolling-budget available to Admins to allocate to Bounties.
 
-```Claim Approved: [@username][checklistName]([taskIds])([amount] Dash)```
+The funds the PO requests from the Dash Network are purely to top-up the Incubator Wallet and increase the overall budget available to Admins to spend.
```

Comment:
"increase" would be more accurate than "top-up", since the budget has no theoretical or practical "top" or capacity.

Reply:
done